recipes/chewbbaca/build.sh

```diff
@@ -0,0 +1,15 @@
+DEST="$PREFIX/bin/chewBBACA"
```

Comment:
Yould it make sense to use PREFIX/lib/ here?

Reply:
Sure, I've made the change.